### DISCUSSION TITLE
Initial Redesign of UI

### DISCUSSION THREAD

**Discussion by yshashi** [dalenguyen/pdfun#26] [Ideas]
I've been playing around with Pdfun and did some UI redesign, and I wanted to share a screenshot to get your feedback and thoughts.
In this iteration, I've focused on modernizing the look and feel while maintaining functionality.

Please take a look at the screenshot and let me know your thoughts. Do you like the direction I'm heading with the redesign? Are there any specific elements you think could be improved or changed?
![image](https://github.com/dalenguyen/pdfun/assets/21971232/02854395-58f3-4cd4-844c-55173c00cc2d)

**Comment by dalenguyen** [comment]
Thanks, @yshashi. It's much better than the original one! 

There're some considerations the application prefers: 

- Prebuilt UIs from PrimeNG such as [MenuBar](https://primeng.org/menubar) or [Avatar](https://primeng.org/avatar) that can help to save development time.
- Tailwind for styling if needed.
- Responsiveness

**Reply by yshashi** [reply]
1. For menu bar I have managed to do it with tailwind at the moment. But I can create an issue to refactor it with Primeng components
2. Yes used tailwind for styling
3. Responsiveness : Yes it’s responsive

**Reply by dalenguyen** [reply]
That's awesome. Feel free to shoot a PR when you're ready :D
